### PR TITLE
AddingPowerSupport_ginkgo-travis.yml

### PR DESCRIPTION
**Pull request by santosh653**
creating ginkgo-travis.yml with proposed changes.

### CODE REVIEW

**Comment by santosh653** [comment]
@seth-priya @gururajrkatti Can you review this PR?

File: travis-ymls/ginkgo-travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - ppc64le
+language: go
+go:
+  - 1.14.x
+  - 1.15.x
+  - tip
+jobs:
+ exclude:
+  - go: 1.14.x
+    arch: ppc64le
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/ginkgo-travis.yml:24]: Reason for excluding 1.14.x? @santosh653

Comment (santosh653) [reply travis-ymls/ginkgo-travis.yml:24]: @Siddhesh-Ghadi This is the repeated PR(I have closed the previous one) I raised after I was not sure how to squash the commits on already raised PR. 
In the earlier PR, Priya had asked me to skip the failure(As there was a failure on ppc64le for go:1.14.x) & run the code for rest, accordingly I have raised new PR with suggested changes.
Kindly let me know if you need more details.,

Thank You!!

Comment (Siddhesh-Ghadi) [reply travis-ymls/ginkgo-travis.yml:24]: In future PRs make sure to add a comment(reason) in the file, if you are excluding a particular job.

**Comment by Siddhesh-Ghadi** [comment]
LGTM @seth-priya

**Pull request merged**